save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt

```diff
@@ -145,9 +145,7 @@ class VulnerabilityMetadataService(
         ): VulnerabilityMetadata = apply {
             summary = entry.summary ?: entry.details?.take(SUMMARY_LENGTH_FROM_DETAILS)?.let { "$it ..." } ?: "Summary not provided"
             details = entry.details ?: "Details not provided"
-            severityNum = entry.severity?.firstOrNull()
-                ?.scoreNum
-                ?.toFloat() ?: 0f
+            severityNum = entry.severity?.firstOrNull()?.let { getScore(it) } ?: 0f
```

Comment:
```suggestion
            severityNum = entry.severity?.firstOrNull()?.let { it.scoreNum?.toFloat() ?: getScore(it) } ?: 0f
```

Reply:
It is not necessary. We decided to always calculate the value from the vector and not take it from the string. But we compare them and log them if they do not match.